### ISSUE TITLE
Feature: Set memory limits for services and auto reload on exceeding them

### ISSUE THREAD

**Issue by MurzNN** [ferdium/ferdium-app#606]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

Sometimes some services consume too much memory (because of bugs in their code or some other reasons), and reloading the service page resolves this. But we should find the needed service before the system gets too laggy, and do the refresh manually.

### Proposed Solution

Will be good to have a feature to automate this routine. Ferdium will monitor memory usage for each service, which has a configured memory limit, and automatically reload it if the limit is reached.

Will be great to have a general limit, applied to all services, and also have a per-service setting too.

### Alternatives Considered

Alternatively, we can somehow auto-detect memory leaks and automate reloading a problematic service, but I have no idea how to implement this. So manually configured per-service limits will be a good solution.

**Comment by ghost** [comment]
i got 32 gb of ram and 80% of that get consumed with ferdium also when ferdium is in background (when i press quit, not when i close the window).
this problem sucks because ferdium is great

**Comment by dmo-odoo** [comment]
I agree. Each service currently uses 600mb of ram. It quickly adds up. Though in my case it might be due to https://github.com/ferdium/ferdium-app/issues/578.

**Comment by devnoname120** [comment]
> i got 32 gb of ram and 80% of that get consumed with ferdium also when ferdium is in background (when i press quit, not when i close the window).
> this problem sucks because ferdium is great

I have the same issue.

My config:

- Ferdium 6.1.0
- macOS Monterey 12.6
- MacBook Air M1 16 Gb (2021)


I can confirm that this issue doesn't happen with [Franz](https://meetfranz.com).

**Comment by nickchomey** [comment]
Likewise for me. It quickly eats up 3-5GB of my 32GB of ram for about 9 services. I find that I'm having to restart Ferdium numerous times per day.

**Comment by assasukasse** [comment]
Same for me, I use ferdium on a 4GB ram laptop and it quickly spirals out of control..
